integration-tests/bats/sql-local-remote.bats

```diff
@@ -11,6 +11,9 @@ make_repo() {
 }
 
 setup() {
+    if [ "$SQL_ENGINE" = "remote-engine" ]; then
+      skip "This test tests remote connections directly, SQL_ENGINE is not needed."
```

Comment:
Not clear why you are skipping it. Shouldn't you unset the env var and allow the test to continue?